irods_consortium_continuous_integration_build_hook.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
+
+import glob
+import multiprocessing
+import optparse
+import os
+import tempfile
+
+import irods_python_ci_utilities
+
+def add_cmake_to_front_of_path():
+    cmake_path = '/opt/irods-externals/cmake3.21.4-0/bin'
```

Comment:
Need to try using a newer CMake when available...

See https://github.com/irods/irods_rule_engine_plugin_python/commit/674f5657e6d5d67c886684c32f9a9236884944c7

Reply:
Eh, nevermind. We can just piggy-back off of the effort done for other repos if this is not needed to build for the currently supported platforms.